charts/langsmith/templates/_helpers.tpl

```diff
@@ -171,8 +171,6 @@ Template containing common environment variables that are used by several servic
       key: clickhouse_tls
 - name: LOG_LEVEL
   value: {{ .Values.config.logLevel }}
-- name: FF_WORKSPACE_SCOPE_ORG_INVITES_ENABLED
```

Comment:
nit unrelated to your PR, maybe all non-conditional env vars should just be at the top of `commonEnv`?

Reply:
fixed, i inteneded to make this one conditional as well which is why imoved it 🤦